CMakeLists.txt

```diff
@@ -109,8 +109,12 @@ set(XSIMD_HEADERS
 )
 
 add_library(xsimd INTERFACE)
-target_include_directories(xsimd INTERFACE $<BUILD_INTERFACE:${XSIMD_INCLUDE_DIR}>
-                                           $<INSTALL_INTERFACE:include>)
+
+target_include_directories(xsimd INTERFACE
+    $<BUILD_INTERFACE:${XSIMD_INCLUDE_DIR}>
+    $<INSTALL_INTERFACE:include>)
+
+target_compile_features(xsimd INTERFACE cxx_std_14)
```

Comment:
Actually we need xsimd to stay compatible with C++11 because of some libraries that depend on `xtensor`. C++14 is required only when the optional support for `xcomplex` class from the `xtl` is enabled.

Reply:
Done @JohanMabille !